src/styles/fonts.tsx

```diff
@@ -1,6 +1,6 @@
-import {Source_Sans_3} from "next/font/google";
+import {Montserrat} from "next/font/google";
 
-export const sourceSans3 = Source_Sans_3({
+export const sourceSans3 = Montserrat({
```

Comment:
If you change the font, you should change the variable name here and in the `layout.tsx` where it is used.
I'll fix it though.

Reply:
Oh right, good point.